docs/spec/api.md

```diff
@@ -1206,7 +1206,7 @@ The registry does not implement the V2 API.
 401 Unauthorized
 WWW-Authenticate: <scheme> realm="<realm>", ..."
 Content-Length: <length>
-Content-Type: application/json; charset=utf-8
```

Comment:
You might want to propose these changes in the OCI distribution-spec repo as well, because that's where the official spec now lives; https://github.com/opencontainers/distribution-spec/blob/master/spec.md

Reply:
@thaJeztah ack, thanks! I'm going to duplicate the spec changes there.